specification/components/examples/DocumentReferenceBundle.json

```diff
@@ -39,7 +39,7 @@
         "content": [
           {
             "attachment": {
-              "url": "https://api.service.nhs.uk/summary-care-record/Bundle?composition.identifier=FA60BE64-1F34-11EB-A2A8-000C29A364EB&composition.subject:Patient.identifier=https://fhir.nhs.uk/Id/nhs-number|9995000180"
+              "url": "https://api.service.nhs.uk/summary-care-record/Bundle?composition.identifier=FA60BE64-1F34-11EB-A2A8-000C29A364EB$composition.subject:Patient.identifier=https://fhir.nhs.uk/Id/nhs-number|9000000009"
```

Comment:
It seems that you accidentally overwritten some recent changes  (see db5a9ec306776f78d2f5fedc2e65321c1f15efa5)

Reply:
Changed to match master apart from the NHS number.